Cargo.toml

```diff
@@ -7,7 +7,7 @@ version = "0.44.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "MIT"
 repository = "https://github.com/libp2p/rust-libp2p"
-keywords = ["peer-to-peer", "libp2p", "networking"]
+keywords = ["peer-to-peer", "libp2p", "networking", "p2p"]
```

Comment:
For the sake of a clean git history, would you mind doing these unrelated changes in separate pull requests?

Reply:
Alright then!